perseo-utils/src/test/java/com/telefonica/iot/perseo/utils/DateTimeUtilsTest.java

```diff
@@ -51,27 +55,33 @@ public void testGetNextSunset() {
     }
 
     /**
-     * Test get milis to next sunrise.
+     * Test get milis to next sunrise
+     * Calculates milliseconds to next sunrise considering current date
+     * Is successful if the resoult is less than 24 hrs
      */
     public void testGetMilisToNextSunrise() {
         System.out.println("getMilisToNextSunrise");
         Calendar calendar = Calendar.getInstance();
         long nextSunrise = DateTimeUtils.getMilisToNextSunrise(calendar, 40.4131699, -3.6830699);
-        assertTrue(calendar.getTimeInMillis() < nextSunrise);
+        assertTrue(nextSunrise < 86400000);
     }
 
     /**
      * Test get milis to next sunset.
+     * Calculates milliseconds to next sunset considering current date
+     * Is successful if the resoult is less than 24 hrs
      */
     public void testGetMilisToNextSunset() {
         System.out.println("getMilisToNextSunset");
         Calendar calendar = Calendar.getInstance();
         long nextSunset = DateTimeUtils.getMilisToNextSunset(calendar, 40.4131699, -3.6830699);
-        assertTrue(calendar.getTimeInMillis() < nextSunset);
+        assertTrue(nextSunset < 86400000);
```

Comment:
I'd suggest this magic number to a constant, for clarity.

(In general, we don't like magic numbers in code, they use to be opaque ;))

Reply:
I totally agree. This methods are a bit complicated to test without constats so I added javadoc on tests to clarify what constants do.

As you suggested I changed the number to a constant.

Fixed in [f7c0319](https://github.com/telefonicaid/perseo-core/pull/146/commits/f7c03191ae7f7175cb8aa7a610ef13c48e4fe2ce)